dll/win32/shell32/wine/brsfolder.c

```diff
@@ -1104,16 +1127,23 @@ static HTREEITEM BrsFolder_FindItemByPidl(browse_info *info, LPCITEMIDLIST pidl,
 {
     TV_ITEMW item;
     TV_ITEMDATA *item_data;
+#ifndef __REACTOS__
     HRESULT hr;
+#endif
 
     item.mask = TVIF_HANDLE | TVIF_PARAM;
     item.hItem = hItem;
     SendMessageW(info->hwndTreeView, TVM_GETITEMW, 0, (LPARAM)&item);
     item_data = (TV_ITEMDATA *)item.lParam;
 
+#ifdef __REACTOS__
```

Comment:
why is this needed?

Reply:
This source code is based on Wine source. For comparison to Wine source, I inserted `#ifdef __REACTOS__` ... `#endif`.

Reply:
`item_data->lpifq` and `pidl` are absolute PIDLs. We cannot compare them with `IShellFolder_CompareIDs`.